include/seqan3/io/sequence_file/output.hpp

```diff
@@ -38,7 +38,7 @@
 #include <seqan3/io/sequence_file/format_fasta.hpp>
 #include <seqan3/io/sequence_file/format_fastq.hpp>
 #include <seqan3/io/sequence_file/format_genbank.hpp>
-#include <seqan3/io/sequence_file/format_sam.hpp>
+#include <seqan3/io/alignment_file/format_sam.hpp>
```

Comment:
This is not pretty, but it can't be helped I guess...

Reply:
no. unless I declare the functions in the `alignment_file/format_sam.hpp` and then define them in a `sequence_file/format_sam.hpp`. But either way, the alignment format is included and an extra header might be confusing...?